### PR TITLE
Update README with Chocolatey and Homebrew instructions

### LINKED ISSUE #1133: Add link to Chocolatey and Homebrew in README

### PR DESCRIPTION
**Pull request by fbiville**
Fixes #1133

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -8,10 +8,30 @@ It includes commands to install Knative in a Kubernetes cluster, and for managin
 
 == Installation of the latest release
 
-Official binary release are available from the link:https://github.com/projectriff/riff/releases[Releases] page. See link:https://projectriff.io/docs/getting-started-with-knative-riff-on-minikube/[Getting started on Minikube] or
+=== Chocolatey
+
+To install the CLI for Windows with link:https://chocolatey.org/[Chocolatey]:
```
Comment (ericbottard) [inline README.adoc]: Should make it clear that this will likely get you the latest *released* version of the cli (ie not master) but apart from that, lgtm

Comment (fbiville) [reply README.adoc]: The Chocolatey section is within `Installation of the latest release`.

**Comment by fbiville** [comment]
@jldec done

**Comment by ericbottard** [comment]
Merged as 025c79d3

**Pull request closed without merging**